sktime/classification/interval_based/tests/test_tsf.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+import pytest
+from sklearn.dummy import DummyClassifier
+from sklearn.model_selection import train_test_split
+from sktime.classification.interval_based import TimeSeriesForest
+from sktime.datasets import load_gunpoint
+
+
+@pytest.mark.parametrize("n_jobs", [1, 4])
+def test_on_gunpoint(n_jobs):
+    np.random.seed(42)
+    X, y = load_gunpoint(return_X_y=True)
+    X_train, X_test, y_train, y_test = train_test_split(X, y)
+
+    tsf = TimeSeriesForest(random_state=42, n_estimators=20, n_jobs=n_jobs)
+    tsf.fit(X_train, y_train)
+
+    n_classes = len(np.unique(y))
+    proba = tsf.predict_proba(X_test)
+
+    assert proba.shape == (
+        X_test.shape[0],
+        n_classes,
+    ), "Incorrect shape of classes probabilities"
+    np.testing.assert_array_equal(
+        np.ones(X_test.shape[0]),
+        np.sum(proba, axis=1),
+        "Incorrect classes probabilities",
+    )
+
+    tsf_score = tsf.score(X_test, y_test)
+
+    dummy = DummyClassifier(strategy="most_frequent")
+    dummy.fit(X_train, y_train)
+
+    dummy_score = dummy.score(X_test, y_test)
+
+    assert tsf_score > dummy_score, "Classifier performs worse than dummy classifier"
+
+    assert tsf_score >= 0.95, "Classifier performs worse than expected"
+
+
+def test_multiproc_determinism():
```

Comment:
Again this seems like a general tests which all our estimators with the `n_jobs` parameter should pass ... perhaps add it here: https://github.com/alan-turing-institute/sktime/blob/master/sktime/tests/test_all_estimators.py

Reply:
OK

Reply:
Just to be sure, these suggestions are optional, feel free to ignore them, happy to focus on TSF only!

Reply:
I think it's a good suggestion. I'll check first that the other classes actually pass this, as making them to do so would be out of scope here.

Reply:
I moved this test to check all of the estimators and it seems that for `StackingForecaster` and `EnsembleForecaster` the `n_jobs` parameter is overwritten in subclass initialization, so is always set to `None`. I have code with the test which fixes this. Should I make another PR for that?

Reply:
Yes, please! Good catch!